### PR TITLE
Simplify "I'm this person" workflow.

### LINKED ISSUE #96: Simplify "I'm this person" flow
Can we simplify the flow for creating a person record for _myself_? (How popular is this use case?)

In particular, auto-fill the following fields:
  Author's name: Person's name
  Status: I'm this person
  Have you personally talked with this person: YES

### PR DESCRIPTION
**Pull request by tingar**
Copy over the users's name and mark them as contacted when they select that
they're filling out the form for themself.

Fixes #96.

### CODE REVIEW

File: app/resources/forms.js [hunk 1]
```diff
@@ -215,6 +215,26 @@ function view_page_loaded() {
...
+  if (event.target.value == 'is_note_author') {
+    var name;
+    var nameFields = document.querySelectorAll('div.name input[name$=name]');
+    if (nameFields.length) {
+      name = nameFields[0].value + ' ' + nameFields[1].value;
+    } else {
+      name = document.querySelector('.header > h1').textContent
+          .replace(/^\s+|\s+$/, '');
+    }
+    $('author_name').value  = name;
```
Comment (gimite) [inline app/resources/forms.js]: Remove one extra whitespace between "value" and "=".

File: app/resources/forms.js [hunk 2]
```diff
@@ -215,6 +215,26 @@ function view_page_loaded() {
...
+
+  $('status').addEventListener('change', on_status_change);
+}
+
+// Copy the name and set author_made_contact when a user fills out the form for
+// themselves.
+function on_status_change(event) {
+  if (event.target.value == 'is_note_author') {
+    var name;
+    var nameFields = document.querySelectorAll('div.name input[name$=name]');
```
Comment (gimite) [inline app/resources/forms.js]: Can we explicitly specify (family|given)_name here?

File: app/resources/forms.js (hunk 1 above)
Comment (tingar) [inline app/resources/forms.js]: Done.

File: app/resources/forms.js (hunk 2 above)
Comment (tingar) [inline app/resources/forms.js]: Done.

**Comment by gimite** [comment]
LGTM. Merged. Thanks!

**Pull request merged**